_tests/phoenix.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+////	Glob Recursive Function
+// Glob Recursively to a Pattern
+function glob_recursive($Pattern, $Flags = 0, $Strip_Underscore = false) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The variable $Return is not named in camelCase.](https://www.codacy.com/app/lewisgoddard/phoenix/file/1807499961/issues/source?bid=2030171&fileBranchId=3071992#l5)